docs/getting-started/installing-regression-games.mdx

```diff
@@ -4,6 +4,110 @@ sidebar_label: 'Installing Regression Games'
+
+Enables game state collection of Unity ECS Entities and their Components.
+Without this extension, the core SDK can still capture GameObject data in projects using ECS but will have no access to ECS Entities.
+Depending on your project and its needs, this extension may not be necessary, but we highly recommend including it
+for full visibility of your game's internal state.
+
+```
+https://github.com/Regression-Games/RGUnityBots?path=src/gg.regression.unity.bots.ecs#v0.0.27
+```
+
+## Authenticate through the Regression Games SDK
+
+Once our core SDK and applicable extensions have been added to your project, close the Package Manager.
+You will be greeted by a prompt for Regression Games login details.
+
+Create a personal account through [the Regression Games website](https://play.regression.gg/signup).
+This step is simple, and will give you access to many
+features such as saving [Gameplay Sessions](../core-concepts/gameplay-sessions/gameplay-sessions-getting-started)
+and creating powerful [Validation Suites](../core-concepts/validation-suites/validation-suites-getting-started).
+
+:::caution
+You can submit your new account credentials through the Unity prompt now,
+but this method is less secure than creating and using an API key.
+We recommend taking an extra moment to follow the section below.
+:::
+
+### Set an API Key
+
+Generate a new API key through your account settings in the Regression Games client, and copy the generated value to your clipboard.
+
+![Generate a new API key](../core-concepts/img/api-keys/create-new-api-key.gif)
+
+:::info
+To learn more about our API keys, see the [Authenticating with API Keys](../core-concepts/authenticating-with-api-keys) section.
+:::
+
+Do not submit any information to the welcome prompt in Unity.
+Instead, close the prompt and navigate to `Edit` > `Project Settings` > `Regression Games`.
+Paste the key you just generated into the `RG API Key` field, then close the project settings.
+
+## Restart Unity
+
+Now is a good time to restart Unity and any code editor you may have open (e.g. Rider or Visual Studio).
+Without this step, Unity and your code editor may not properly resolve your new packages even after Unity recompiles the scripts.
+
+## Add the Regression Games Overlay to Your Scene
+
+Our core package provides an in-game overlay that acts as a UI for features such as performing
+[input recording and playback](../core-concepts/gameplay-sessions/gameplay-sessions-getting-started)
+and constructing and running [Bot Sequences](../core-concepts/bot-sequences/getting-started-with-bot-sequences-and-segments),
+but it also drives our data collection behind-the-scenes.
+This means the overlay prefab **must** be present in each scene where you want to use any Regression Games features.
+However, you can hide the visibility of the overlay through the project settings under
+`Edit` > `Project Settings` > `Regression Games` if needed.
+
+Find the `RGOverlayCanvasV2` prefab by navigating to
```

Comment:
We'll want to use the `RGOverlayCanvas` without the `V2`, it has the new Sequences stuff

Reply:
Good call. Since we're using an mvp branch to avoid releasing these docs until everything is ready I'll make sure to capture info for the upcoming changes rather than what we have public now